doc/callbacks.md

```diff
@@ -8,53 +8,74 @@ as:
 
 Callbacks can be passed as an argument to the `Optimize()` function:
 
+<details>
+<summary>Click to check out the example</summary>
+<p>
+
 ```c++
-RosenbrockFunction f;
-arma::mat coordinates = f.GetInitialPoint();
+  RosenbrockFunction f;
+  arma::mat coordinates = f.GetInitialPoint();
 
-MomentumSGD optimizer(0.01, 32, 100000, 1e-5, true, MomentumUpdate(0.5));
+  MomentumSGD optimizer(0.01, 32, 100000, 1e-5, true, MomentumUpdate(0.5));
 
-// Pass the built-in *PrintLoss* callback as the last argument to the
-// *Optimize()* function.
-optimizer.Optimize(f, coordinates, PrintLoss());
+  // Pass the built-in *PrintLoss* callback as the last argument to the
+  // *Optimize()* function.
+  optimizer.Optimize(f, coordinates, PrintLoss());
 ```
 
+</p>
+</details>  
+
 Passing multiple callbacks is just the same as passing a single callback:
 
+<details>
+<summary>Click to check out the example</summary>
```

Comment:
Another thought, do you think we can change this text to `Click to collapse/expand example code.`?  That way the message makes sense even when it's expanded.

Reply:
Agreed, ``Click to collapse/expand example code.`` is more descriptive.

Reply:
I also vote for `Click to collapse/expand example code`

Reply:
Done :) :rocket: